src/components/ModalWindow/index.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import styles from './modal-window.scss';
+import settings from 'settings';
+import CloseButton from 'components/CloseButton';
+
+const Modal = React.createClass({
```

Comment:
I'm not sure if this thing isn't just too smart for a dumb component. Shouldn't some logic be extracted and moved to [`ModalWindowContainer`](https://github.com/iris-dni/iris-frontend/blob/feature/181-signin-overlay/src/containers/ModalWindow.js)?